test/org/sagebionetworks/bridge/models/studies/ConsentSignatureTest.java

```diff
@@ -207,6 +308,41 @@ public void jsonHappyCaseWithImage() throws Exception {
     }
     
     @Test
+    public void signatureContainsSignedOnValue() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+        JsonNode jsonNode = JSON_OBJECT_MAPPER.readTree(json);
+
+        ConsentSignature sig = ConsentSignature.createFromJson(jsonNode, UNIX_TIMESTAMP);
+        assertEquals(UNIX_TIMESTAMP, sig.getSignedOn());
+    }
+    
+    @Test
+    public void existingSignatureJsonDeserializesWithoutSignedOn() throws Exception {
+        String json = "{\"name\":\"test name\",\"birthdate\":\"1970-01-01\"}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(json, ConsentSignature.class);
+        assertEquals("test name", sig.getName());
+        assertEquals("1970-01-01", sig.getBirthdate());
+        assertEquals(0, sig.getSignedOn());
```

Comment:
Yes, I'm adding that, this works well. Just verifying there's a test that proves the value is kept on deserialization. I'm realizing it the signedOn value will have to go all the way through the UserConsentDao on the next merge, it has to be used when the UserConsent3 record is created.